docs/user/installation_and_configuration.rst

```diff
@@ -136,7 +141,7 @@ Copying the runtime JAR to each tablet server as above will work, but in
 Accumulo 1.6+, we can leverage namespaces to isolate the GeoMesa classpath
 from the rest of Accumulo.
 
-To install the distributed runtime JAR, use the ``install-geomesa-namespace.sh``
+To install the distributed runtime JAR(s), use the ``install-geomesa-namespace.sh``
```

Comment:
Presumably this script needs to be updated to have an option for copying the raster jar.